napalm/eos/eos.py

```diff
@@ -542,9 +542,9 @@ def extract_temperature_data(data):
         if not is_veos:
             for psu, data in power_output['powerSupplies'].items():
                 environment_counters['power'][psu] = {
-                    'status': data['state'] == 'ok',
-                    'capacity': data['capacity'],
-                    'output': data['outputPower']
+                    'status': data.get('state', False) == 'ok',
```

Comment:
Looking on what we did on Junos (and somewhat on IOS), I think we should probably have status default to True in the unknown case)...i.e. think this is probably more consistent with other drivers and we have no basis for saying there is a problem. Also the -1 will indicate we don't have data.